error_handling_on_chain/README.md

```diff
@@ -0,0 +1,127 @@
+# Error Handling On Chain
+
+This example shows you how to handle errors in Blocky AS function calls on chain.
+It builds on the [Hello World On Chain](../hello_world_on_chain) example,
+which shows you how to verify and parse a Blocky AS attestation on chain.
+
+In this example, you'll learn how to:
+
+- Write a smart contract to extract the `Result` from a function attestation
+- Check `Result` for an error to determine whether to return the output or to
+revoke the transaction
+
+## Setup
+
+Set up the project dependencies:
+
+```bash
+npm install
+```
+
+## Quick Start
+
+To run this example, call:
+
+```bash
+make test-local
+```
+
+which will test extracting and verifying `Result` from an attested fn call in
+[`contracts/User.sol`](contracts/User.sol) 
+within a local test environment:
+
+```
+  Local Test
+        Success: true
+        Error: 
+        Value: {"number":42}
+    ✔ Verify TA and parse Result w/success (616ms)
+        Success: false
+        Error: expected error
+        Value: null
+    ✔ Verify TA and parse Result w/error (189ms)
+```
+
+## Walkthrough
+
+We cover the basics of how to verify an attested function call in a smart
+contract in the
+[Hello World - Bringing A Blocky AS Function Call Attestation On Chain](../../hello_world_on_chain)
+example.
+
+In this example, we go a step further and show you how to extract the `Result`
+struct from the `successFunc` and `errorFunc` function call attestations in the
+[Error Handling](../error_handling) example.
```

Comment:
Maybe add that we copied the attestations resulting from those functions into the `inputs` folder